.github/workflows/feature-serverless-framework-deployment.yml

```diff
@@ -72,10 +72,10 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
           aws-region: us-west-1
 
-      - name: Set up Go 1.19
+      - name: Set up Go 1.21
         uses: actions/setup-go@v2
         with:
-          go-version: 1.19
+          go-version: 1.21
```

Comment:
Could you add hellogo deployment to the e2e pipeline?

Reply:
`      - name: AWS end-to-end test
        working-directory: ${{env.working-directory}}
        run: ./main --o latency-samples --c ../experiments/tests/aws/hellogo.json --s true`
Do you mean something like the above? Or similar to what you did in #272 to have the pipeline deploy it using the Serverless framework?